docs/lifecycle-events.md

```diff
@@ -25,7 +25,9 @@ Fired before the element is removed from the DOM.
 Note that when using this event you are responsible for removing the element yourself.
 
 ```js
-element.parent.removeChild(element)
+if (element.parentNode) {
```

Comment:
Right, onremove is not called when elements are replaced. This has been poorly discussed before and no agreement has been reached yet. 😅